lib/commands/update.js

```diff
@@ -114,7 +114,9 @@ class UpdateCommand extends Command {
                     return;
                 }
 
-                return this.ui.listr(tasks, context);
+                return this.ui.listr(tasks, context).catch(error =>
+                    this.rollbackFromFail(error, context.version, argv['auto-rollback'])
```

Comment:
We have to differentiate between

1. errors which are migration/ghost errors
2. errors which happened in the CLI

Reply:
```if (error instanceof errors.GhostError)```

Reply:
Also: we should not ask to rollback if you have executed `ghost update --rollback`.